### DISCUSSION TITLE
Files not visible in Minio after ingestion through postman

### DISCUSSION THREAD

**Discussion by rohandixitdelhi95** [Sunbird-cQube/community#834] [General]
After ingesting event file through postman for new program student attendance, files are not reflecting in Minio bucket.
Can you suggest possible ways to debug the issue.

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha Please check the ingestion logs and send me the screenshot. please find the below command to check the ingestion logs.
sudo docker logs ingest_app

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha please follow the below document and do the ingestion again .

https://docs.google.com/document/d/1eoElNSMm9g_IwmWBVFDmfRfYgRdalNxG/edit

**Reply by rohandixitdelhi95** [reply]
please give me the access to the doc

**Comment by rohandixitdelhi95** [comment]
@bommalla 
please check the logs

<img width="960" alt="image" src="https://github.com/Sunbird-cQube/community/assets/140483814/e4698260-923e-454a-b391-b4b45e6dee5f">
<img width="960" alt="image" src="https://github.com/Sunbird-cQube/community/assets/140483814/7bf8b977-7e78-47f9-bfbc-5f1604fc1331">

**Comment by bommalla** [comment]
@rohandixitdelhi95 @RNarmatha 
Please  check the data,I think date format is wrong. please use  dd/mm/yy format.
ex: 02/11/23

**Comment by RNarmatha** [comment]
@rohandixitdelhi95 - Any updates on this issue. Was the issue fixed ?
